pam_yubico.c

```diff
@@ -164,8 +169,19 @@ authorize_user_token (struct cfg *cfg,
 		      pam_handle_t *pamh)
 {
   int retval = AUTH_ERROR;
-
-  if (cfg->auth_file)
+  if (cfg->mysql_server)
+    {
+      #ifdef HAVE_MYSQL
```

Comment:
please move this to the leftmost column

Reply:
Sorry for the delay, I haven't see your comment.

done in the last commit.